db/migrate/20170720194715_create_middleware_jdr_reports.rb

```diff
@@ -0,0 +1,12 @@
+class CreateMiddlewareJdrReports < ActiveRecord::Migration[5.0]
+  def change
+    create_table :middleware_jdr_reports do |t|
```

Comment:
@Fryguy I renamed the table. Please, confirm if a model is needed in main repo. If so, I'll add columns to enable STI.